sphinx/source/docs/user_guide/styling.rst

```diff
@@ -90,7 +90,7 @@ Text Properties
 
 .. include:: ../includes/text_props.txt
 
-.. _userguide_styling_colors:
+.. _userguide_visible_property:
```

Comment:
FYI the scheme I had been trying to use for sphinx references was
```
_<chapter>_<section>
_<chapter>_<section>_<subsection> here
```
So I'd suggest `_userguide_styling_visible_property`

Reply:
I edited the file because something was off with the anchors. I think ``_userguide_styling_colors:`` somehow got moved to the wrong place... Your suggested name for the new anchor makes a lot of sense, it'll be included in my next commit!